### PR TITLE
Minor fixes for Cli Output

### PR DESCRIPTION
**Pull request by piotr-cz**
- Used Regex backreference when looking up style tags (See [Example #2 at php.net](php.net/manual/en/function.preg-match-all.php#refsect1-function.preg-match-all-examples))
- Used Regex [DOTALL modifier](http://php.net/manual/pl/reference.pcre.pattern.modifiers.php) for multi-line strings
- Code style

### CODE REVIEW

**Comment by elkuku** [comment]
Looks good :wink:

**Comment by piotr-cz** [comment]
not to Travis.

Who says `JRegistryFormatXMLTest::testObjectToString Failed asserting that two strings are equal.`
The actual test result is missing a [XML Prolog](https://github.com/joomla/joomla-framework/blob/staging/src/Joomla/Registry/Tests/format/FormatXmlTest.php#L23) `<?xml version="1.0"?>`. But how does it relate to this?

**Comment by elkuku** [comment]
It's not related and pops up on all PRs currently here as well as on the CMS.
There is an "approach" to fix it - here: https://github.com/joomla/joomla-cms/pull/2596

**Comment by piotr-cz** [comment]
I see, thanks for info

File: src/Joomla/Application/Cli/ColorProcessor.php [hunk 1]
```diff
@@ -29,7 +49,7 @@ class ColorProcessor
 	 * @param   string      $name   The style name.
 	 * @param   ColorStyle  $style  The color style.
 	 *
-	 * @return $this
+	 * @return  $this
```
Comment (eddieajau) [inline src/Joomla/Application/Cli/ColorProcessor.php]: This line should read:

```
@return  ColorProcessor  Returns itself to support chaining.
```

**Comment by piotr-cz** [comment]
@eddieajau Should be fine now.

**Comment by dbhurley** [comment]
Can you update this to latest so it can be merged?

**Comment by piotr-cz** [comment]
Updated. Most of the changes have been committed in meantime by #288 which caused merge conflicts.

**Pull request merged**